gtsfm/evaluation/metrics_report.py

```diff
@@ -0,0 +1,198 @@
+import os
+from typing import Any, Dict, List, Tuple, Union
+
+import plotly.graph_objects as go
+import plotly.subplots as psubplot
+from tabulate import tabulate
+
+from gtsfm.evaluation.metrics import GtsfmMetric, GtsfmMetricsGroup
+
+SUBPLOTS_PER_ROW = 3
+SUMMARY_KEY = "summary"
+DATA_KEY = "full_data"
+
+
+def get_readable_metric_name(metric_name: str):
+    """Helper to convert a metric name separated by underscores to readable format.
+
+    In readable format, each word is capitalized and are separated by spaces.
```

Comment:
maybe a brief example could help in the docstring, like "translation_error_deg" to "Translation Error Deg."

Reply:
Thanks for adding this